src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.Windows.cs

```diff
@@ -44,7 +44,14 @@ private bool LoadCalendarDataFromSystemCore(string localeName, CalendarId calend
             }
 
             // If running using ICU on Windows we should honor user overrides using NLS and the rest from ICU
-            bool result = IcuLoadCalendarDataFromSystem(localeName, calendarId);
+            bool result =
```

Comment:
this is .Windows file, does it need to be changed ?

Reply:
I am not sure. I added it because it was using ICU function: `IcuLoadCalendarDataFromSystem` that won't work without static data that we removed. `CalendarData.Widnows` and `CalendarData.Unix` files are added unconditionally to `System.Private.CoreLib.Shared.projitems` but let's see if removal will break something.